en/docs/tutorials/scenarios/scenario1-create-rest-api.md

```diff
@@ -1,4 +1,75 @@
 # Scenario 1 - Create REST API from an OpenAPI Definition
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Coltrain is one of the railway companies that is partnered up with GOGO Train to provide better service to their customers. Coltrain already has some internally managed APIs deployed in-house and these are managed by their internal development team. One of the APIs is train schedule retrieval API which is intended for the public community to get the Coltrain schedules. Currently, this API is exposed for the public and Coltrain development team faces challenges in maintaining and handling high load for the API . By exposing this API through WSO2 API Manager, Coltrain expects to get the full benefits of an API Management solution such as API lifecycle management, security, throttling, etc. and decouple the maintenance overhead from the internal teams.**
+
+**_Time to Complete : 10 mins_**
+
+WSO2 API manager provides capability to import OAS definitions and create the API using that. 
+    ![Import OAS definition to API-M]({{base_path}}/assets/img/tutorials/scenarios/oas-def-import.png)
+    
+Let's first test the backend. You could do following curl command and you will get a response with schedule items.
+
+```
+curl "http://localhost:8082/train-operations/v1/schedules"
+
+```
+You will get the following response
+
+```
+[{"entryId":"1","startTime":"14:50","endTime":"19:59","from":"London","to":"Glasgow","trainType":"Standard"},{"entryId":"2","startTime":"14:30","endTime":"19:20","from":"London","to":"Edinburgh","trainType":"1st class"},{"entryId":"3","startTime":"11:10","endTime":"12:59","from":"London","to":"Cardiff","trainType":"Standard"},{"entryId":"4","startTime":"08:30","endTime":"10:50","from":"London","to":"Manchester","trainType":"1st class"}]
+
+```
+
+Backend is working fine. Let’s expose this API through WSO2 API Manager. 
+
+1. Log in to https://localhost:9443/publisher/apis using Coltrain publisher user. Use user as _apiprovider@coltrain.com_ and password as _user123_
```

Comment:
```suggestion
1. Log in to https://localhost:9443/publisher/apis using a Coltrain publisher user. Use the following user credentials:

   * username: `_apiprovider@coltrain.com_`
   * password as `_user123_`
```

Reply:
Fixed. used the same format as the other docs in the tutorial